components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -119,109 +120,127 @@ class SitePermissionsFeature(
         session: Session,
         permissionRequest: PermissionRequest
     ): DoorhangerPrompt {
-        return if (!permissionRequest.containsVideoAndAudioSources()) {
-            val prompt = handlingSingleContentPermissions(session, permissionRequest)
-            prompt
+        val context = anchorView.context
+        val uri = Uri.parse(permissionRequest.uri ?: "")
+        val host = " ${uri.host}"
```

Comment:
Why do we need the space in front of the host? 🤔

Reply:
Good catch, I didn't notice it!